tests/test-cases.ts

```diff
@@ -0,0 +1,45 @@
+// these are used by both Playwright and Cypress.
+// They're stored here to make sure we test the same things with both frameworks
+// and don't get out of alignment on what we support
+export const testCases = [
+  {
+    title: 'Asset Paths / query params determine which asset is served',
+    path: 'asset-paths/query-params',
+  },
+  {
+    title: 'Asset Paths / asset doesnt prevent directory from being created',
+    path: 'asset-paths/asset-at-directory-name',
+  },
```

Comment:
I'm really happy with the reuse between test frameworks. But if we want to actually assert things in the tests (e.g. click a button, verify that some text shows), we'd need to add those things here as properties (like `lookForText: 'I am a title'`) and translate them in Playwright and Cypress to the actual framework-specific assertions. Doesn't seem too crazy, but let me know if you feel that's too abstracted. 

We're not doing any assertions in our tests, but eventually I assume we'll want to to make the tests more real-world.